packages/client/tests/functional/0-legacy-ports/malformed-id/tests.ts

```diff
@@ -24,7 +25,7 @@ testMatrix.setupTestSuite(
         /client/tests/functional/0-legacy-ports/malformed-id/tests.ts:0:0
 
            XX testMatrix.setupTestSuite(
-           XX   () => {
+          XX   () => {
```

Comment:
random removed white-space 🤔 probably not related to the PR itself?

Reply:
It is related and needed here actually.

Reply:
But it's really annoying ...